### ISSUE TITLE
Aufgaben einer Klausur deaktivierbar machen

### ISSUE THREAD

**Issue by BeneLuWi** [BeneLuWi/examanager_v3#9]
Eine Aufgabe sollte eine (optionale) Liste mit Klassen haben für die die Aufgabe aus der Bewertung herausfliegt.

## Backend
- [ ] `Task`-Modell anpassen
- [ ] Berechnung anpassen
- [ ] Optionale Berechnung mit deaktivierten Aufgaben zB Query Parameter `include_deactivated`
## Frontend
- [ ] Anzeige in der UI (Eingabe und Darstellung) hinzufügen
- [ ] Vergleich der Statistiken mit Deaktivierung

**Comment by DavidKaub** [comment]
Sollen die Ergebnisse (nicht die Statistik) in der Excel Datei auch entsprechend ausgeschlossen bzw. nur bei Flag `include_deactivated_tasks` berücksichtigt werden oder gilt das Deaktivieren der Tasks nur für die Statistik?

**Comment by BeneLuWi** [comment]
Ja, ich würde die deaktivierte Aufgabe aus allen Berechnungen rausnehmen. Allerdings wäre es cool, wenn die erreichten Punkte trotzdem in der Excel stehen würden, allerdings mit einem Tag im Spaltennamen: `gestrichen` oder so etwas. 

Vllt kann man die Spalte mit pandas auch noch einfärben oder so? (Das scheint zu gehen, sieht aber nicht mega trivial aus ^^)
